example-mojaloop-backend/values.yaml

```diff
@@ -426,65 +274,38 @@ ttksims-redis:
   ##
   architecture: standalone
 
+  ## @param fullnameOverride String to fully override  template
+  ##
+  fullnameOverride: ttksims-redis 
+
   ## Redis&reg; Authentication parameters
   ## ref: https://github.com/bitnami/containers/tree/main/bitnami/redis#setting-the-server-password-on-first-run
   ##
-  auth:
-    ## @param auth.enabled Enable password authentication
-    ##
-    enabled: false
```

Comment:
Why?

Reply:
@mdebarros : Interestingly I was defaulting the top level enabled: true for the bitnami charts but I think having this as a local example makes sense and I have put it back in at the top level for all of the charts